final release/format_data.py

```diff
@@ -0,0 +1,208 @@
+
```

Comment:
Need a file-level docstring.

Reply:
I'll ask for you guidance in this

Reply:
https://www.python.org/dev/peps/pep-0257/#specification